buddyboss/buddyboss-platform
# PROD-705 #3111 - Fix: Issue with someone leaves the group but not being shown in message notification

https://buddyboss.atlassian.net/browse/PROD-705

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3111 .

### How to test the changes in this Pull Request:

1. Try logging in to one user that is a member of the group.
2. Try leaving the group and switching the user to another member of the group.
3. You can see that message notification count is being added and can see that message notification.

**NOTE:** There are few other cases to test this issue:
1. Join a new member
2. Left member
3. Banned member
4. Unbanned member
5. Delete the Group Messages from the particular user account


### Proof Screenshots or Video
https://vimeo.com/646849758/e99eadb29b


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
